fe/fe-core/src/main/java/com/starrocks/common/util/PrintableMap.java

```diff
@@ -40,6 +40,8 @@ public class PrintableMap<K, V> {
         SENSITIVE_KEY.add("bos_secret_accesskey");
         SENSITIVE_KEY.add("fs.s3a.access.key");
         SENSITIVE_KEY.add("fs.s3a.secret.key");
+        SENSITIVE_KEY.add("aws.s3.access_key");
```

Comment:
Also gcs/azure/oss/cos?

Reply:
All authentication listed in our document has been covered now.